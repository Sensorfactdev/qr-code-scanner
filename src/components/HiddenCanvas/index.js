import styled from 'styled-components';

export default styled.canvas`
  display: none;
`;
