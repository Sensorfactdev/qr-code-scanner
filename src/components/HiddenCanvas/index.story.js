import React from 'react';
import { storiesOf } from '@storybook/react';
import HiddenCanvas from './index';

storiesOf('HiddenCanvas', module)
  .add('default view', () => (
    <HiddenCanvas />
  ));
