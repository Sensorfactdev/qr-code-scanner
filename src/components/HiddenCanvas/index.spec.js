import React from 'react';
import renderer from 'react-test-renderer';
import HiddenCanvas from './index';

describe('HiddenCanvas', () => {
  it('should render', () => {
    const component = renderer.create(
      <HiddenCanvas />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
