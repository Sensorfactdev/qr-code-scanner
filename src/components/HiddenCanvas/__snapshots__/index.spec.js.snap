// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HiddenCanvas should render 1`] = `
<div
  className="sc-bdVaJa fGYvF"
/>
`;
